doc/setup/native_libs.md

```diff
@@ -14,47 +14,116 @@ The key element for calling the GDAL native library from mapfishapp is the **ima
  * a GDAL Java native binding library, based on the JNI framework, named gdaljni, or ogrjni,
  * and the GDAL library.
 
-The latter can be installed, on Debian-based distributions, with the libgdal1 package:
+### Install GDAL from Debian Jessie packages : 
 
-    sudo apt-get install libgdal1
+The latter can be installed, from Debian Jessie distributions, with some libgdal packages:
+```
+    apt-get install libgdal1h libgdal-java
+
+```
+
+Then you will have to make sure that the webapps access to `gdal.jar` (/usr/share/java/gdal.jar).
+To do that, when your app is deployed, copy your `gdal.jar` in the **WEB-INF/lib** folder of the webapp, and delete `imageio-ext-gdal-bindings-*.jar`. Think about include this in your deploy script.
+
+Notice : this method doesn't allow to have multiple apps with gdal in the same tomcat, but actually we can't get shared classloader to work because gdal load the native and imageio also load them, there is a conflict.
+
+And finally edit your `/etc/default/tomcat8` file to have something like :
+```
+LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+JAVA_OPTS="${JAVA_OPTS} -Djava.library.path=$LD_LIBRARY_PATH"
+```
+
+### Install GDAL from sources
+
+If you do not want to use the precompiled binaries, another way to install the GDAL Java binding is building it from sources.
+
+The advantage to compile GDAL is to allow more formats to be supported (ecw, jp2K, ...), you can choose the ones which suits your needs.
+
+In order to compile we'll need some compiler packages :
+```
+apt-get install build-essential
+```
 
-Some more work is needed for installing the GDAL Java binding library, as there is still no deb package for it (note that packages exist for ruby and perl bindings, hopefully the Java's one will be released soon - see a recent proposal http://ftp-master.debian.org/new/gdal_1.10.0-0%7Eexp3.html).
+#### Enable ECW support
```

Comment:
Mention that this is an optional step, and that carefully crafted GeoTiff pyramids are as fast as ECW.